app/templates/components/forms/order-form.hbs

```diff
@@ -0,0 +1,181 @@
+<form class="ui form" autocomplete="off" {{action 'submit' on='submit' }}>
```

Comment:
I didn't get you. Here `submit` is button type, not the text which will be seen to the user and adding translation helper is leading to test fail